solara/__main__.py

```diff
@@ -361,16 +361,19 @@ def run(
     #         # window.show()
     #         webview.start(debug=True)
 
-    def open_browser():
-        while not failed and (server is None or not server.started):
+    def check_server_started():
+        while not failed and (server is None or not server.started) or not solara.server.server.is_ready(url):
             time.sleep(0.1)
         if not failed:
-            webbrowser.open(url)
+            # remove the Solara server is starting ... line and print the url (in green)
+            print(f"\r\x1b[1;32mSolara server is running at {url}\x1b[0m")
```

Comment:
Would it be nicer to leave the `Solara server is starting...` in, and print `Solara server started at {url}` in green after?

Reply:
yeah, maybe the \r can be removed.